index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Sin Titulo s01</title>
```

Comment:
El titulo apropiado seria el siguiente
```suggestion
    <title>Pagina principal</title>
```